keyboards/melgeek/z70ultra/rev1/rules.mk

```diff
@@ -13,7 +13,9 @@ MOUSEKEY_ENABLE = yes          # Mouse keys
 EXTRAKEY_ENABLE = yes          # Audio control and System control
 CONSOLE_ENABLE = no            # Console for debug
 COMMAND_ENABLE = no            # Commands for debug and configuration
-NKRO_ENABLE = yes           # Enable N-Key Rollover
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
```

Comment:
Please revert this change

Reply:
The change probably happened unintentionally, does it affect the project?